packages/p2p/src/components/app.jsx

```diff
@@ -69,124 +128,87 @@ const App = observer(props => {
-                            </div>
-                        )}
-                        {should_show_verification && general_store.is_advertiser && <Download />}
-                        {!should_show_verification && (
-                            <Tabs
-                                onTabItemClick={general_store.handleTabClick}
-                                active_index={general_store.active_index}
-                                className={classNames('p2p-cashier', {
-                                    'p2p-cashier__advertiser-tab': is_mobile && general_store.is_advertiser,
-                                })}
-                                top
-                                header_fit_content
-                            >
-                                <div label={localize('Buy / Sell')}>
-                                    <BuySell navigate={general_store.redirectTo} params={general_store.parameters} />
-                                </div>
-                                <div count={general_store.notification_count} label={localize('Orders')}>
-                                    <Orders
-                                        navigate={general_store.redirectTo}
-                                        params={general_store.parameters}
-                                        chat_info={general_store.chat_info}
-                                    />
-                                </div>
-                                <div label={localize('My ads')}>
-                                    <MyAds navigate={general_store.redirectTo} params={general_store.parameters} />
-                                </div>
-                                {general_store.is_advertiser && (
-                                    <div label={localize('My profile')}>
-                                        <MyProfile
-                                            navigate={general_store.redirectTo}
-                                            params={general_store.parameters}
-                                        />
-                                    </div>
-                                )}
-                            </Tabs>
-                        )}
-                    </>
+        <P2pWrapper {...wrapper_props}>
+            <Tabs
+                onTabItemClick={general_store.handleTabClick}
+                active_index={general_store.active_index}
+                className='p2p-cashier__tabs'
+                top
+                header_fit_content={!isMobile()}
+                is_100vw={isMobile()}
+            >
+                <div label={localize('Buy / Sell')}>
+                    <BuySell />
+                </div>
+                <div count={general_store.notification_count} label={localize('Orders')}>
+                    <Orders
+                        navigate={general_store.redirectTo}
+                        params={general_store.parameters}
+                        chat_info={general_store.chat_info}
```

Comment:
What about these?

Reply:
The `Orders` component still expects these props.